core-libs/setup/ssr/error-handling/multi-error-handlers/server-responding-error-handler.ts

```diff
@@ -26,10 +29,11 @@ export class ServerRespondingErrorHandler implements MultiErrorHandler {
   protected propagateServerErrorResponse = inject(
     PROPAGATE_SERVER_ERROR_RESPONSE
   );
-  protected isSsrErrorHandlingEnabled = inject(ENABLE_SSR_ERROR_HANDLING);
+  protected isServerErrorPropagationEnabled =
+    inject(FeatureToggles).serverErrorPropagation;
```

Comment:
Let's:
- inject FeatureConfigService as `private` property
- call it inline .isEnabled() in line 36